application/dash_application/views/single_lane_rna.py

```diff
@@ -71,13 +71,16 @@
 special_cols = {
     "Total Reads (Passed Filter)": "Total Reads PassedFilter",
```

Comment:
Is Total Reads still needed for anything?

Reply:
Total Reads are still in the table.